Gemfile

```diff
@@ -1,10 +1,10 @@
 source "https://rubygems.org"
 
 # You can run against local Maze Runner branches and uncommitted changes:
-gem 'bugsnag-maze-runner', path: '../maze-runner'
+#gem 'bugsnag-maze-runner', path: '../maze-runner'
 
 # Or a specific release:
-#gem 'bugsnag-maze-runner', git: 'https://github.com/bugsnag/maze-runner', tag: 'v3.2.0'
+gem 'bugsnag-maze-runner', git: 'https://github.com/bugsnag/maze-runner', tag: 'v3.5.1'
```

Comment:
If you wait until later this morning you can do straight to v3.6.0, which will enforce the presence of the Bugsnag-Integrity header on all requests (not just that the digest is correct if present)